### DISCUSSION TITLE
HowTo Use blueCFD with QtCreator?

### DISCUSSION THREAD

**Discussion by likui** [blueCFD/Core#189] [Q&A]
HowTo Use blueCFD with QtCreator?

**Comment by wyldckat** [comment]
The closest we are aware of is this tutorial: https://openfoamwiki.net/index.php/HowTo_Use_OpenFOAM_with_QtCreator
The concept is the same, the missing trick should be that QtCreator should be launched from the command line.

Instructions on how to add the path for another application into blueCFD-Core's terminal: https://bluecfd.github.io/Core/FAQ/how-to-install-additional-software-such-as-gmsh/

**Reply by likui** [reply]
Hi wyldckat, 
Thank you for your reply.
I added the path of qtCreator into blueCFD-Core's terminal, launched qtCreator from the command line, imported from simpleFoam, configured for build settings. 
When I build the project, I got following error. 

`
08:36:49: 为项目simpleFoam执行步骤 ...
08:36:49: 正在启动 "C:\PROGRA~1\BLUECF~1\OpenFOAM-8\wmake\wmake.bat" 


C:\Program Files\blueCFD-Core-2020\OpenFOAM-8\applications\solvers\incompressible\simpleFoam>sh.exe -c "wmake " 
$(C:/PROGRA~1/BLUECF~1/OpenFOAM-8/wmake/scripts/makeReinterpretExePath x86_64-w64-mingw32-g++) -std=c++11 -Dmingw_w64 -DWM_ARCH_OPTION=64 -DWM_DP -DWM_LABEL_SIZE=32 -DWIN64 -DLITTLE_ENDIAN -DWIN64 -DLITTLE_ENDIAN -Wall -Wextra -Wold-style-cast -Wnon-virtual-dtor -Wno-unused-parameter -Wno-invalid-offsetof -Wno-attributes -O2 -DNDEBUG -gdwarf  -DNoRepository -ftemplate-depth-100 -D_FILE_OFFSET_BITS=64 -D_MODE_T_ -IC:/PROGRA~1/BLUECF~1/OpenFOAM-8/src/MomentumTransportModels/momentumTransportModels/lnInclude -IC:/PROGRA~1/BLUECF~1/OpenFOAM-8/src/MomentumTransportModels/incompressible/lnInclude -IC:/PROGRA~1/BLUECF~1/OpenFOAM-8/src/transportModels/lnInclude -IC:/PROGRA~1/BLUECF~1/OpenFOAM-8/src/finiteVolume/lnInclude -IC:/PROGRA~1/BLUECF~1/OpenFOAM-8/src/meshTools/lnInclude -IC:/PROGRA~1/BLUECF~1/OpenFOAM-8/src/sampling/lnInclude -DLIBS_TO_LOAD="\"momentumTransportModels,incompressibleMomentumTransportModels,incompressibleTransportModels,finiteVolume,meshTools,fvOptions,sampling\"" @Make/mingw_w64GccDPInt32Opt/includeHeaderPaths -IlnInclude -I. -IC:/PROGRA~1/BLUECF~1/OpenFOAM-8/src/OpenFOAM/lnInclude -IC:/PROGRA~1/BLUECF~1/OpenFOAM-8/src/OSspecific/MSwindows/lnInclude   -c simpleFoam.C -o C:/Program Files/blueCFD-Core-2020/OpenFOAM-8/applications/solvers/incompressible/simpleFoam/Make/mingw_w64GccDPInt32Opt/simpleFoam.o

X8E3D3~1.EXE: error: Files/blueCFD-Core-2020/OpenFOAM-8/applications/solvers/incompressible/simpleFoam/Make/mingw_w64GccDPInt32Opt/simpleFoam.o: No such file or directory
`

**Reply by wyldckat** [reply]
I forgot about this... Check this wiki page for the missing steps for setting up a build environment: https://github.com/blueCFD/Core/wiki/Loose-notes-on-building-custom-code

**Reply by likui** [reply]
Thank you very much.
